autopilot/autopilot.py

```diff
@@ -126,8 +127,17 @@ def init(configuration, options, plugin):
     plugin.num_channels = int(options['autopilot-num-channels'])
     plugin.percent = int(options['autopilot-percent'])
     plugin.min_capacity_sat = int(options['autopilot-min-channel-size-msat']) / 1000
+    plugin.initialized = threading.Event()
+    plugin.autopilot = None
+    print('Initialized autopilot function')
 
-    plugin.autopilot = CLightning_autopilot(plugin.rpc)
+    def intialize_autopilot():
```

Comment:
There is a typo in the method name `intialize_autopilot`, this should be `initialize_autopilot` so the Thread.init call couple of lines below can start it.